apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <html lang="<%= Configuration.locale %>">
 <head>
-  <title><%= content_for?(:title) ? yield(:title) : "Dashboard - #{OodAppkit.dashboard.title}" %></title>
-  <%= favicon_link_tag 'favicon.ico', href: OodAppkit.public.url.join('favicon.ico'), skip_pipeline: true %>
+  <title><%= content_for?(:title) ? yield(:title) : "Dashboard - #{@user_configuration.dashboard_title}" %></title>
+  <%= favicon_link_tag 'favicon.ico', href:@user_configuration.public_url.join('favicon.ico'), skip_pipeline: true %>
```

Comment:
```suggestion
  <%= favicon_link_tag 'favicon.ico', href: @user_configuration.public_url.join('favicon.ico'), skip_pipeline: true %>
```

Reply:
I'm now seeing this - `skip_pipeline: true` - which relates to our other CSS ticket. I'd bet you can add css files in a public folder you'd just need this flag.

Reply:
will update